release/pkg/manager/calico/manager.go

```diff
@@ -328,14 +340,34 @@ func (r *CalicoManager) BuildMetadata(dir string) error {
 		return err
 	}
 
-	err = os.WriteFile(fmt.Sprintf("%s/metadata.yaml", dir), []byte(bs), 0o644)
+	err = os.WriteFile(filepath.Join(dir, metadataFileName), []byte(bs), 0o644)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
+func (r *CalicoManager) getRegistryFromManifests() (string, error) {
```

Comment:
Don't we have this function defined somewhere else already? Or am I misremembering?

Reply:
this function was in closed source